volumes/rescu/README.md

```diff
@@ -1,45 +1,316 @@
+>`raw_15min` table:
+"dt", 
+"raw_info", 
+"raw_uid"
 
-1) The script reads information from a .rpt file and then inserts the date into a table named `rescu.raw_15min`. The table has the following information
+>`volumes_15min` table:
+"detector_id",
+"datetime_bin",
+"volume_15min",
+"arterycode",
+"volume_uid",
 
-|raw_uid|dt|raw_info|
-|--|--|--|
-|12852|2020-01-13|1700 - 1715 de0010deg    633|
-|12853|2020-01-13|1700 - 1715 de0010der    62|
 
-2) There is also a trigger function named [`rescu.insert_rescu_volumes()`](create-trigger-function-populate_volumes_15min.sql) which processes the newly added data from `rescu.raw_15min` and inserts the processed data into the table `rescu.volumes_15min`. All information from `raw_info` in the raw table is then processed into the 3 columns which are `detector_id`, `datetime_bin` and `volume_15min` whereas `aretrycode` is taken from the table `rescu.detector_inventory` by matching them using `detector_id`. The table `rescu.volumes_15min` has the following information
+## Missing data: 
+> TBA
 
-|volume_uid|detector_id|datetime_bin|volume_15min|arterycode|
-|--|--|--|--|--|
-|9333459|DE0010DEG|2020-01-13 17:00:00|	749|	23984|
-|9333460|DE0010DER|2020-01-13 17:00:00|	80|	3272|
 
-## `check_rescu.py`
 
-Since the terminal server does not have an internet connection, we will not get notified if the process fails. Therefore, we created an Airflow task to do that job for us. There is a dag named [`check_rescu`](/dags/check_rescu.py) which runs at 6am everyday that checks the number of rows inserted for the day before in both tables `rescu.raw_15min` and `rescu.volumes_15min`. If the number of rows is 0 OR if the number of rows from the raw table is less than that in the volumes table OR if the total number of rows from the volumes table is less than 7000 (the average number of rows per day is about 20,000), a Slack notification will be sent to notify the team. The line that does exactly that is shown below.
-```python
-if raw_num == 0 or raw_num < volume_num or volume_num < 7000:
-  raise Exception ('There is a PROBLEM here. There is no raw data OR raw_data is less than volume_15min OR volumes_15min is less than 7000 which is way too low')
-```
+## Specialized formats or other abbreviations used:
+>TBA
```

Comment:
Let's hide (or "comment out") headings that don't have info. They'll stay in the template but they won't be visible in the preview.
To comment out multi-line stuff do this:

```
<!--- # I am a heading
I am the information that goes under the heading --->

```

Reply:
Done